### ISSUE TITLE
Question about the limit of package and weight

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9903]
![img_v2_ec756667-5d04-4cc4-bfa5-c5a7ac14bf6l](https://github.com/user-attachments/assets/cb2c9d8e-003c-47d8-bbc7-4a17ac52659d)
When we call the **/shipment-management/shipments/create-commands/{commandId}**, commandId:4ee34bc5-2e8f-4a07-beff-197edefdff30, there is a error returned that the returned  the size exceeded the limit. Package size: 64 x 41 x 38 cm
Weight: 20kg

**Comment by yangcen6** [comment]
Does this restriction apply to all carriers? Since we are selling TVs, how can we avoid this problem?

**Comment by Lukasz-Zurek** [comment]
Yes, but depending on the carrier, dimensions may vary. We do not return this type of information in the API. It is worth testing and researching.
